docs/fast_data/runtime_management.mdx

```diff
@@ -0,0 +1,147 @@
+---
+id: runtime_management
+title: Runtime Management
+sidebar_label: Runtime Management
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+
+# Runtime Management
+
+Welcome to Runtime Management - your tool for achieving better control over the Fast Data environment during runtime.
+This platform empowers you to dynamically manage the execution of Fast Data, offering functionalities to pause and resume microservices effortlessly.
+Dive into the world of streamlined runtime control and take charge of your Fast Data deployment with ease.
+
+## Overview
+
+Runtime Management is specifically designed to facilitate the runtime deployment of Fast Data.
+It provides a user-friendly web interface and robust backend functionalities to enhance your control over the Fast Data microservices environment.
+
+With Runtime Management, users gain the ability to govern potentially every aspect of the Fast Data runtime.
+Currently, this powerful mechanism is supported in the Single View Creator, offering more than just the suspension of Single View generation on MongoDB.
+
+### Key Features and Benefits
+
+- **Flexible Control:** govern the execution of Fast Data microservices with fine-grained control, allowing for strategic management of the runtime environment.
+
+- **Enhanced Performance and Consistency:** utilize Runtime Management to optimize performance and ensure data consistency. 
+      For example, start the ingestion of a massive data system into the Fast Data environment. Only after the ingestion process is complete, 
+      initiate the processing of updates from the Real Time Updater (and the Single View Trigger Generator if active) to generate Single Views. 
+      This sequential approach enhances performance and data consistency.
+
+- **Strategic Ingestion:** achieve better performance and consistency of data by strategically managing the ingestion process, aligning it with the broader data processing pipeline.
+
+### Useful Scenarios
```

Comment:
I would merge this paragraph with the previous one, trying to better define the use case. Some of the options in this paragraph are similar to each other and some of them can be considered as sub-topic of the topics in the previous paragraph